docs/IE_DG/ONNX_Support.md

```diff
@@ -9,10 +9,41 @@ InferenceEngine::Core core;
 auto network = core.ReadNetwork("model.onnx");
 ```
 
+**Reshape feature:**
+
 OpenVINO™ doesn't provide a mechanism to specify pre-processing (like mean values subtraction, reverse input channels) for the ONNX format.
 If an ONNX model contains dynamic shapes for input, please use the `CNNNetwork::reshape` method for shape specialization.
 
-Unsupported types of tensors:
+**Weights saved in external files:**
+
+OpenVINO™ supports ONNX models which use weights saved in external files. It is especially useful for models larger than 2GB because of protobuf limitations.
```

Comment:
```suggestion
OpenVINO™ supports ONNX models that use weights saved in external files. It is especially useful for models larger than 2GB because of protobuf limitations.
```

Reply:
You can also say "which store weights in external files"

Reply:
done